lua_zlib.c

```diff
@@ -1267,7 +1267,7 @@ LUALIB_API int luaopen_zlib(lua_State * const L) {
 
     SETLITERAL("_COPYRIGHT", "Copyright (c) 2009-2016 Brian Maher");
     SETLITERAL("_DESCRIPTION", "Simple streaming interface to the zlib library");
-    SETLITERAL("_VERSION", "lua-zlib $Id$");
+    SETLITERAL("_VERSION", "lua-zlib $Id: 9dc7bcd71c3994bc4c8e9dc86e5beaca440de17f $");
```

Comment:
This may or may not be a legitimate change (I'm not commenting on the release workflow or where these substitutions happen, although this might happen on checkout and hence not be something to commit) but surely this does not quality as a "spelling mistake".

Reply:
Oh, sorry, my bad. I'll fix.